api/src/main/resources/liquibase.xml

```diff
@@ -342,6 +352,14 @@
         <sqlFile path="patientUpcomingAppointments.sql"/>
     </changeSet>
 
+    <changeSet id="global-property-upcoming-appointments-sql-update" author="Shankar, Mahesh">
```

Comment:
add global property in config.xml to add global property. 
example: https://github.com/Bahmni/openmrs-module-appointments/blob/master/omod/src/main/resources/config.xml#L61

Reply:
This is adding value in the table name global_property and not a global property used in java code.